jbpm-installer/src/main/resources/build.properties

```diff
@@ -40,11 +40,11 @@ jBPM.console.url=https://origin-repository.jboss.org/nexus/service/local/artifac
 # for EAP7:
 #   jBPM.casemgmt.url=https://origin-repository.jboss.org/nexus/content/groups/public-jboss/org/jbpm/jbpm-wb-case-mgmt-showcase/${jBPM.version}/jbpm-wb-case-mgmt-showcase-${jBPM.version}-eap7.war
 # for WildFly19:
-#   jBPM.casemgmt.url=https://origin-repository.jboss.org/nexus/content/groups/public-jboss/org/jbpm/jbpm-wb-case-mgmt-showcase/${jBPM.version}/jbpm-wb-case-mgmt-showcase-${jBPM.version}-wildfly19.war
+#   jBPM.casemgmt.url=https://origin-repository.jboss.org/nexus/content/groups/public-jboss/org/jbpm/jbpm-wb-case-mgmt-showcase/${jBPM.version}/jbpm-wb-case-mgmt-showcase-${jBPM.version}-wildfly23.war
```

Comment:
we should replace origin repository whenever it is still referenced I think

Reply:
DONE